jbt-it/neomdb
# 176 client create project application

How to test:

- go to http://localhost:3000/projekte
- click on "bewerben"
- check if all fields are working as expected
- check if the "bewerben" button is disabled if no availability is selected, no restriction is specified if availability "Mit Einschränkungen" is selected and if the secrecy agreement is not checked
